uwblueprint/jump-math
# feat: better assessment editor loading state

## Notion ticket link

[Improve save behaviour on admin assessments](https://www.notion.so/uwblueprintexecs/Improve-save-behaviour-on-admin-assessments-1dc65e756a794d4bafdab43b80bac0ff)



## Implementation description
- Add full-page whiteout instead of loading spinner
- Handle reload-prompt properly



## Steps to test
1. Open an assessment in the editor
2. Click back and verify there is no prompt
3. Open it again
4. Edit something
5. Click back and verify there is a prompt and hit cancel
6. Click save
7. Verify there is a full-screen whiteout box and you can't click any elements
8. After saving finishes, click back
9. Verify there is no prompt
10. Repeat the edit-save cycle once more
11. This time, instead of clicking back, edit a field again
12. Click back and verify you see a prompt again



## What should reviewers focus on?
* 


## Checklist
